pkg/asset/installconfig/openstack/openstack.go

```diff
@@ -55,7 +55,7 @@ func Platform() (*openstack.Platform, error) {
 			Prompt: &survey.Select{
 				Message: "Region",
 				Help:    "The OpenStack region to be used for installation.",
-				Default: "regionOne",
+				Default: "RegionOne",
```

Comment:
I believe we should instead remove the default value as it will take precedence over whatever comes from regionNames.
regionNames is populated with the actual region names for a given cloud.

Reply:
ok, I have updated this to remove the default value ..